packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -369,7 +369,7 @@ export class VSXExtensionEditorComponent extends AbstractVSXExtensionComponent {
                             {this.renderNamespaceAccess()}
                             {publisher}
                         </span>
-                        {downloadCount && <span className='fa fa-download download-count' onClick={this.openExtension}>{downloadFormatter.format(downloadCount)}</span>}
+                        {!!downloadCount && <span className='fa fa-download download-count' onClick={this.openExtension}>{downloadFormatter.format(downloadCount)}</span>}
                         {averageRating && <span className='average-rating' onClick={this.openAverageRating}>{this.renderStars()}</span>}
```

Comment:
could you replace here with `averageRating !== undefined`, so in the case of 0 we still show 5 stars